Tone/signal/Pow.ts

```diff
@@ -0,0 +1,85 @@
+import { Signal, SignalOptions } from "./Signal";
+import { WaveShaper, WaveShaperMappingFn } from "./WaveShaper";
+import { optionsFromArguments } from "Tone";
+
+/**
+ *  @class Pow applies an exponent to the incoming signal. The incoming signal
+ *         must be AudioRange.
+ *
+ *  @constructor
+ *  @param exp The exponent to apply to the incoming signal, must be at least 2. 
+ *  @example
+ * var pow = new Pow(2);
+ * var sig = new Signal(0.5).connect(pow);
+ * //output of pow is 0.25. 
+ */
+export class Pow extends Signal<number> {
+	
+	readonly name: string = "Pow";
+
+	/**
+	 * Indicates if the value should be overridden on connection
+	 */
+	readonly override = false;
+	
+	/**
+	 * The exponent
+	 */
+	private _exponent!: number;
+	
+	private _exponentScaler = new WaveShaper({
```

Comment:
As a small optimization you can move this class constructor into the `constructor()` function which will save setting the exponent function twice, once when it's created and another time when the exponent is set.